tools/plist_to_html/plist_to_html/PlistToHtml.py

```diff
@@ -213,7 +213,9 @@ def create(self, output_path: str, report_data: ReportData):
         # Add severity levels for reports.
         for report in report_data['reports']:
             checker = report['checkerName']
-            report['severity'] = self._severity_map.get(checker, 'UNSPECIFIED')
+            report['severity'] = 'UNSPECIFIED'
+            if self._checker_labels:
+                report['severity'] = self._checker_labels.severity(checker)
```

Comment:
Lets create a function to get severity level for a cheker in this class and use it in both places:
```py
def get_severity(self, checker):
    """ TODO """
    if not self._checker_labels:
        return 'UNSPECIFIED'

    return self._checker_labels.severity(checker)

or

def get_severity(self, checker):
    """ TODO """
    return self._checker_labels.severity(checker) \
        if self._checker_labels else 'UNSPECIFIED'
```